data/data/bootstrap/baremetal/files/etc/containers/systemd/ironic.container.template

```diff
@@ -26,6 +26,9 @@ Environment="INSPECTOR_HTPASSWD=${IRONIC_HTPASSWD}"
 Environment="IRONIC_KERNEL_PARAMS=${IRONIC_KERNEL_PARAMS}"
 Environment="HTTP_PORT=${HTTP_PORT}"
 
+# TODO: Remove after https://review.opendev.org/c/openstack/ironic/+/905626 merges
```

Comment:
nit: we can actually keep this one, the raw images code path is useless for the installer

Reply:
Comment removed